cmd/vicadmin/vicadm.go

```diff
@@ -311,8 +318,11 @@ func httpEntry(name string, res *http.Response) (entry, error) {
 
 func (path urlReader) open() (entry, error) {
 	defer trace.End(trace.Begin(string(path)))
-
-	res, err := http.Get(string(path))
+	timeout := time.Duration(2 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
```

Comment:
I'm just saying 2 seconds is too optimistic as a timeout, and I' m guessing it's just a random number you picked. Note that timeout is not a connection establishment time but that includes everything including reading the body.

Bump it to some sane worst case (yet another random number but bigger than 2), like 60 or 600 as  it won't affect the normal/expected case, it's just for bounding the worst case.